lua/nvim-treesitter/parsers.lua

```diff
@@ -254,10 +254,11 @@ list.java = {
 
 list.kotlin = {
   install_info = {
-    url = "https://github.com/Joakker/tree-sitter-kotlin",
-    files = { "src/parser.c" },
+    url = "https://github.com/fwcd/tree-sitter-kotlin",
+    branch = "main",
+    files = { "src/parser.c", "src/scanner.c" },
   },
-  maintainers = { "@Joakker" },
+  maintainers = { "@fwcd" },
```

Comment:
you would be the maintainer. it's not about maintenance of the upstream repo but its integration into this repo

Reply:
Sorry about that, I misunderstood.